js/ui/select_box.js

```diff
@@ -501,7 +501,7 @@ const SelectBox = DropDownList.inherit({
         this.callBase();
     },
 
-    _restoreInputText: function() {
+    _restoreInputText: function(saveEditingValue) {
```

Comment:
Do we need to take into account the `saveEditingValue` argument on the TagBox side?

Reply:
Thanks, it is a good idea. The TagBox does not hat have the initial issue, but we can unify its behavior with dxSelectBox